spec.bs

```diff
@@ -4557,14 +4557,17 @@ dictionary StorageInterestGroup : AuctionAdInterestGroup {
 
 *This first introductory paragraph is non-normative.*
 
-{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an <dfn>auction nonce</dfn>, a
-one-time [=version 4 UUID=] uniquely associated with a single call to
-{{Window/navigator}}.{{Navigator/runAdAuction()}}. For multi-seller auctions, this ID is uniquely
-associated with all {{AuctionAdConfig/componentAuctions}}. This nonce will need to be passed back in
-via a subsequent call to {{Window/navigator}}.{{Navigator/runAdAuction()}} via the
-{{AuctionAdConfig}}. This is currently only needed for [=auctions=] that use [=additional bids=],
-for which the [=auction nonce=] will be included in each [=additional bid=] as a way of ensuring
-that those bids are only used in the [=auctions=] for which they were intended.
+{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an <dfn>auction nonce</dfn>, which
+is a one-time canonical [=string=] representation of a [=version 4 UUID=] that is uniquely
+associated with a single call to {{Window/navigator}}.{{Navigator/runAdAuction()}}. For multi-seller
+auctions, a (separate) auction nonce can be uniquely associated with each of the
+{{AuctionAdConfig/componentAuctions}}. The auction nonce(s) will need to be passed back in via a 
+subsequent call to {{Window/navigator}}.{{Navigator/runAdAuction()}} via the {{AuctionAdConfig}}.
+This is currently only needed for [=auctions=] that use [=additional bids=], for which the auction
+nonce will be included in each [=additional bid=] as a way of ensuring that those bids are only
+used in the [=auctions=] for which they were intended. Alternatively, when using a [=signed

```

Comment:
Can we flip this around so that the seller nonce case is the default behavior, and the auction nonce directly included in the additional bid is the "exceptional" behavior? Perhaps ...

```
This is currently only needed for [=auctions=] that use [=additional bids=], in which the auction nonce
is combined with a [=signed additional bid with metadata/seller nonce=] to construct a bid nonce
that must be included in each [=additional bid=]. For backwards compatibility, [=additional bids=]
may include an auction nonce directly in place of a bid nonce.
```